docs/aerodromes/Cairns.md

```diff
@@ -74,10 +72,9 @@ d) Aircraft departing to the Western VFR Corridor from all arriving aircrafton o
 
 IFR aircraft shall be processed via one of the following SIDs:
 
-a) RWY 15, Jets via SWIFT: SWIFT SID. Non-Jets via NONUM: NONUM
-SID
-b) RWY 33, All Jets: EAZEE SID, Radar Transition
-c) All others: Cairns SID(R)
+a) RWY 15, Jets via SWIFT: SWIFT SID. Non-Jets via NONUM: NONUM SID  
+b) RWY 33, All Jets: EAZEE SID, Radar Transition  
+c) All others: CS RADAR SID  
```

Comment:
Small thing, but in some places you've written "BN (RADAR) SID", and some you've written "CS RADAR SID" up to you I guess which one you go with